src/corona_hakab_model/util.py

```diff
@@ -32,6 +32,17 @@ def upper_bound(d):
     return d.b + d.kwds.get("loc", 0)
 
 
+def get_numpy_uniform_dist(l_bound=None, u_bound=None):
```

Comment:
i think you should not allow l_bound to be None because it must have a value either way

Reply:
sorry for all the comments... i know i'm annoying

Reply:
I thought about furhet extending this (to not giving a but do giving b)
I decieded it is not worth the effort, so done :)